packages/docusaurus-theme-classic/src/theme-classic.d.ts

```diff
@@ -354,6 +354,16 @@ declare module '@theme/CodeBlock' {
   export default function CodeBlock(props: Props): JSX.Element;
 }
 
+declare module '@theme/CodeInline' {
+  export interface Props {
+    readonly children: string;
```

Comment:
The `CodeBlock` component is a lot more complex in this regard and allows elements within code. I think however string-only should be sufficient for inline code.